src/src/modules/auth/auth.service.ts

```diff
@@ -103,17 +103,16 @@ export class AuthService {
 			});
 		}
 	}
-	public async resetPasswordUsingOtp(req, response) {
-		console.log('req', req);
+	public async getUserByusername(req) {
```

Comment:
add one blank line before start of new method

Reply:
done